deck_of_cards/lib/deck_of_cards.rb

```diff
@@ -0,0 +1,17 @@
+class DeckOfCards
+  attr_reader :deck
+
+  def initialize
+    suit = %w( C D H S )
+    rank = %w( Ace King Queen Jack 10 9 8 7 6 5 4 3 2 )
+    @deck = rank.product(suit).map(&:join)
```

Comment:
I'll rewrite this.  I wanted to challenge myself and do this with a Struct to learn how to use them better which I'll work on and resubmit.